src/nomad_material_processing/vapor_deposition/cvd/general.py

```diff
@@ -59,7 +70,7 @@
 )
 
 configuration = config.get_plugin_entry_point(
-    'nomad_material_processing.vapor_deposition.cvd:schema',
+    'nomad_material_processing.vapor_deposition.cvd:movpe_schema',
```

Comment:
Why does the `nomad_material_processing.vapor_deposition.cvd.general` import the config for `nomad_material_processing.vapor_deposition.cvd.movpe`?